requirements.txt

```diff
@@ -12,11 +13,10 @@ requests
 click
 transformers
 readerwriterlock
---extra-index-url https://download.pytorch.org/whl/cu113
 torch
 torchvision
-torchaudio 
-colossalai
+torchaudio
 omegaconf
+hiq-python
```

Comment:
Why we need hiq?

Reply:
Oh. I was using `hiq` to benchmark the fastapi servers in examples and planning to submit several more PRs after this one were merged. One example is:

![image](https://user-images.githubusercontent.com/94136485/226413806-c86330e3-de09-4c14-8cbc-01799e1ef934.png)

I tried to revert to remove `hiq` but found I need to write many tedious functions to do the same thing, so I kept it there, also considering it is a very friendly library :-).

If you think we should remove it, I will come back two days later to remove it because I am really busy on something else right now. Thank you.